test/bigint.jl

```diff
@@ -305,11 +305,6 @@ end
 @test_throws InexactError floor(BigInt,Inf)
 @test_throws InexactError ceil(BigInt,Inf)
 
-@test bin(big(3)) == "11"
-@test oct(big(9)) == "11"
-@test oct(-big(9)) == "-11"
-@test hex(big(12)) == "c"
```

Comment:
Where else are the single-argument versions of these being tested? I'd rather leave these